message_sender/tests.py

```diff
@@ -2017,6 +2017,51 @@ def test_send_voice_override_payload_multiple_urls(self):
         self.assertEqual(r['nested_data']['unknown'], 'unknown')
         self.assertEqual(r['filename'], ['test1.mp3', 'test2.mp3'])
 
+    @responses.activate
+    def test_send_voice_strip_fielpath_language(self):
```

Comment:
the function name, did you maybe mean "filepath" instead of "fielpath"?

Reply:
Haha thanks, good catch!